test/integration/model/import/assemblerFileReader/AssemblerFileReaderCollectionTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019  (original work) Open Assessment Technologies SA;
+ *
+ * @author Oleksandr Zagovorychev <zagovorichev@gmail.com>
+ */
+namespace oat\taoDeliveryRdf\test\integration\model\import\assemblerFileReader;
+
+
+use common_exception_Error;
+use oat\generis\test\MockObject;
+use oat\generis\test\TestCase;
+use oat\oatbox\filesystem\File;
+use oat\taoDeliveryRdf\model\import\assemblerDataProviders\assemblerFileReaders\AssemblerFileReaderCollection;
+use oat\taoDeliveryRdf\model\import\assemblerDataProviders\assemblerFileReaders\AssemblerFileReaderInterface;
+use tao_models_classes_service_StorageDirectory;
+
+class AssemblerFileReaderCollectionTest extends TestCase
```

Comment:
This test looks like unit test (everything is mocked, we don't use file system or db) and should be in `unit` folder.